StoreKitUnitTests/StoreKitConfigTestCase.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreKitConfigTestCase.swift
+//
+//  Created by Andrés Boedo on 23/9/21.
+
+import Foundation
+import Nimble
+@testable import RevenueCat
+import StoreKitTest
+import XCTest
+
+@available(iOS 14.0, tvOS 14.0, macOS 11.0, watchOS 6.2, *)
+class StoreKitConfigTestCase: XCTestCase {
```

Comment:
we can use this as a base class for these unit tests, so that we don't have to do the setup manually in each file